service/resolvers/catalog/fieldsResolver.ts

```diff
@@ -1,7 +1,8 @@
 import axios from 'axios'
 import {IOContext} from 'colossus'
-import {compose, evolve, juxt, map, omit, path, pick, prop, propOr, toPairs} from 'ramda'
+import {compose, evolve, juxt, map, omit, path, pick, prop, propOr, toPairs, assoc, last, split} from 'ramda'
 import paths from './../paths'
+import slugify from 'slugify'
```

Comment:
You need to use `import * as slugify from 'slugify'`.

Take a look here: https://github.com/Microsoft/TypeScript/issues/16093

Fortunately Typescript 2.7 will behave just like Babel and we won't have to worry about that anymore.

Reply:
knew I was missing something 
thanks for the tip